opm/simulators/linalg/getQuasiImpesWeights.hpp

```diff
@@ -87,59 +96,74 @@ namespace Amg
         return weights;
     }
 
-    template<class Vector, class GridView, class ElementContext, class Model>
-    void getTrueImpesWeights(int pressureVarIndex, Vector& weights, const GridView& gridView,
-                             ElementContext& elemCtx, const Model& model, std::size_t threadId)
+    template<class Vector, class ElementContext, class Model, class ElementChunksType>
+    void getTrueImpesWeights(int pressureVarIndex, Vector& weights,
+                             ElementContext& elemCtx,
```

Comment:
Since this is not actually written to, it should be a const reference. Also in ...Analytic().